src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -147,6 +147,12 @@ public void Initialize(TestLoggerEvents events, string testRunDirectory)
             events.TestRunMessage += this.TestMessageHandler;
             events.TestResult += this.TestResultHandler;
             events.TestRunComplete += this.TestRunCompleteHandler;
+
+            // Register for the discovery events.
+            events.DiscoveryMessage += this.TestMessageHandler;
+
+            // TODO Get changes from https://github.com/Microsoft/vstest/pull/1111/
```

Comment:
Should there be a tracking item for this?

Reply:
I have reopened the [PR](https://github.com/Microsoft/vstest/pull/1111/)